src/main/java/seedu/address/logic/parser/TogglePrivacyCommandParser.java

```diff
@@ -59,6 +59,11 @@ public TogglePrivacyCommand parse(String args) throws ParseException {
         if (argMultimap.getValue(PREFIX_RATING).isPresent()) {
             epp.setPrivateRating(false);
         }
+
+        if (!epp.anyNonNullField()) {
```

Comment:
Handled the probability that user might not enter any fields, and prompt users with commandexception.